script.js

```diff
@@ -0,0 +1,120 @@
+    /* eslint-disable */
+    function applyFunction() {
+      let testString = document.getElementById("testString").value;
+      let selectMethod = document.getElementById("selectMethod").value;
+      let error = '';
+      let result;
+      let methodDetails = '';
```

Comment:
@andela-sayidu has this now been resolved but is somehow not reflecting here 😕 ?